### DISCUSSION TITLE
Procedure entry point SystemParametersInfoForDpi could not be located

### DISCUSSION THREAD

**Discussion by gbakeman** [chris2511/xca#515] [Q&A] [answered]
Just had a chance to upgrade from 2.4.0 to 2.5.0. Initially, I attempted to do an in-place upgrade with the MSI installer. Upon launching the application, I get an error message with the title _xca.exe - Entry Point Not Found_. The body reads:

> The procedure entry point `SystemParametersInfoForDpi` could not be located in the dynamic link library C:\Program Files\xca\Qt6Gui.dll.

The only option is to click OK and allow the application to terminate.

I've already attempted to completely uninstall the application then reinstall it using the same downloaded MSI, which results in the same error. I'm not sure how I'm the only one who's seen this error so far... am I missing something obvious?

OS: Windows Server 2008 R2

**Comment by chris2511** [comment, marked as answer]
Looks like `Windows Server 2008` is not supported by Qt6:
https://learn.microsoft.com/de-de/windows/win32/api/winuser/nf-winuser-systemparametersinfofordpi#requirements
https://doc.qt.io/qt-6/supported-platforms.html#windows

You may try the XCA-Qt5 build: https://github.com/chris2511/xca/actions/runs/6292220598
and report, whether it works.

**Comment by gbakeman** [comment]
Hi @chris2511 ,

The Qt5 build does indeed solve the issue. So if my understanding is correct, then by the Qt system requirements, XCA's system requirements are Windows 10 Build 1809 or later?
